impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -96,10 +97,10 @@ static size_t AppendToExistingAtlas(
       return i;
     }
     glyph_positions.push_back(Rect::MakeXYWH(
-        location_in_atlas.x(),                      //
-        location_in_atlas.y() + height_adjustment,  //
-        glyph_size.width,                           //
-        glyph_size.height                           //
+        location_in_atlas.x() + 1,                      //
```

Comment:
I don't follow this adjustment. Perhaps a short comment? If I had to guess, perhaps to account for errors in sample locations.

Reply:
I see the correction to the position later. But still not sure why?

Reply:
This is part of the glyph padding so that we can avoid clearing the atlas. Though I didn't check in this PR, eventually we should be able to remove the clear.